pom.xml

```diff
@@ -37,10 +37,10 @@
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<version.httpcomponents.fluent.hc>4.5.13</version.httpcomponents.fluent.hc>
 		<version.httpcomponents.httpclient>4.5.13</version.httpcomponents.httpclient>
-		<version.httpcomponents.httpcore>4.4.13</version.httpcomponents.httpcore>
-		<version.commons.io>2.5</version.commons.io>
-		<version.commons.codec>1.11</version.commons.codec>
-		<version.jackson.databind>2.9.10.7</version.jackson.databind>
+		<version.httpcomponents.httpcore>4.4.15</version.httpcomponents.httpcore>
+		<version.commons.io>2.11.0</version.commons.io>
+		<version.commons.codec>1.15</version.commons.codec>
```

Comment:
Similar to the comment for `httpcore`, there's no vulnerability for this library. Plus, the httpclient dependency is relying on the older `codec` version.

Reply:
Same as the httpcore, the task was to bump everything, the release notes shows no potential problems and we manage it, if not to bump i suggest to drop the menagement.

Reply:
The `httpcore` and the `commons.codec` dependencies are explicitly defined because we do direclty use classes from the 2 libraries, not to achieve custom version management.
After some second thoughts and after checking the release notes I'm fine with the bump of the dependencies, but I will wait for a second opinion on the matter.